setup.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+from setuptools import setup, find_packages
+import glob
+
+templates = glob.glob("templates/*")
+model_spec = glob.glob("acis_thermal_check/*_check/*.json")
+
+entry_points = {}
+entry_points['console_scripts'] = [
+    'dea_check = acis_thermal_check.dea_check.dea_check:main',
+    'dpa_check = acis_thermal_check.dpa_check.dpa_check:main',
+    'psmc_check = acis_thermal_check.psmc_check.psmc_check:main',
+]
+
+
+setup(name='acis_thermal_check',
+      packages=find_packages(),
+      version='1.0.0',
+      description='ACIS Thermal Model Library',
+      author='John ZuHone',
+      author_email='jzuhone@gmail.com',
+      url='http://github.com/acisops/acis_thermal_check',
+      download_url='https://github.com/acisops/acis_thermal_check/tarball/1.0.0',
+      data_files=[('templates', templates)],
```

Comment:
Having `templates` as non-package data might not work so well with the standard Ska installation.  If I do `python setup.py sdist` then the resultant source tarball does not have those data files (yay Python packaging).  The Ska distribution is maintained via source tarballs in /proj/sot/ska/pkgs.

If you were to install from the git repo in the way we do for Ska (essentially `pip install .`) then there is a different issue because the `templates` directory files get put into `$prefix/templates` which is not localized to your package.  You can see this in action more easily by going to the git repo and doing:
```
mkdir local
export PYTHONPATH=$PWD/local/lib/python2.7/site-packages
python setup.py install --prefix=local --old-and-unmanageable  # don't use eggs
find local
...
local/lib/python2.7/site-packages/acis_thermal_check-1.0.0-py2.7.egg-info/SOURCES.txt
local/bin
local/bin/psmc_check  # OK
local/bin/dpa_check
local/bin/dea_check
local/templates  # NOT OK
local/templates/acis_thermal_check.css
local/templates/html4css1.css
local/templates/index_template.rst
local/templates/index_template_val_only.rst
```
(We don't use eggs when possible because they don't work with PYTHONPATH which makes testing harder).

So I *think* the best thing would be to just move `templates` into `acis_thermal_check` and probably include the files in `MANIFEST.in`, but I haven't tried this.